runtime-modules/versioned-store/Cargo.toml

```diff
@@ -10,14 +10,14 @@ codec = { package = 'parity-scale-codec', version = '1.3.1', default-features =
 sp-std = { package = 'sp-std', default-features = false, git = 'https://github.com/paritytech/substrate.git', rev = '00768a1f21a579c478fe5d4f51e1fa71f7db9fd4'}
 frame-support = { package = 'frame-support', default-features = false, git = 'https://github.com/paritytech/substrate.git', rev = '00768a1f21a579c478fe5d4f51e1fa71f7db9fd4'}
 system = { package = 'frame-system', default-features = false, git = 'https://github.com/paritytech/substrate.git', rev = '00768a1f21a579c478fe5d4f51e1fa71f7db9fd4'}
+common = { package = 'pallet-common', default-features = false, path = '../common'}
```

Comment:
Add `'common/std'` line to the `std` feature.

Reply:
done in 9d3d637e976c721efefc72a36cb8e5fdf2217160